src/textual/suggester.py

```diff
@@ -136,3 +136,19 @@ async def get_suggestion(self, value: str) -> str | None:
             if suggestion.startswith(value):
                 return self._suggestions[idx]
         return None
+
+
+class SuggestLastWordFromList(SuggestFromList):
+    """
+        Give suggestions for completing each last word in the text based on a fixed list of options.
+    """
+
+    async def get_suggestion(self, value: str) -> str | None:
+        if not value.endswith(' '):
+            values = value.rsplit(' ', 1)
+            last_word = values[-1]
+            for idx, suggestion in enumerate(self._for_comparison):
```

Comment:
Probable shouldn't be using "private" data here. Suggest directly suggesting from `Suggester`.